02-spatial-data.Rmd

```diff
@@ -986,7 +989,8 @@ The same function, `projection()`, is used to set a CRS for raster objects.
 The main difference, compared to vector data, is that raster objects only accept `proj4` definitions:
 
 ```{r}
-projection(new_raster) = "+proj=utm +zone=12 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs" # set CRS
```

Comment:
At 110 characters this line is definitely too long, thanks for spotting this. And the code still seems to work! Many thanks.